compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2609,7 +2615,11 @@ TR::Register *OMR::Power::TreeEvaluator::vstoreEvaluator(TR::Node *node, TR::Cod
    switch(node->getDataType())
      {
      case TR::VectorInt8:
+   cg->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P9) ? opcode = TR::InstOpCode::stxvb16x : opcode = TR::InstOpCode::stxvw4x;
+   break;
      case TR::VectorInt16:
+   cg->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P9) ? opcode = TR::InstOpCode::stxvh8x : opcode = TR::InstOpCode::stxvw4x;
+   break;
```

Comment:
This is an interesting issue. If we disable support for byte/short store/load on LE before P9 it will essentially disable all Vector API support for those types on these platforms. Maybe better to keep it the way @midronij implemented it but disable operations that will be affected by incorrect order? For example, when we start implementing getting vector element by its index then we will either disable that operation for byte/short before P9 or will adjust getting vector element correspondingly.

Reply:
In that case, this PR should be ready to merge. I'll open a separate issue for the P8 and lower case